android/src/main/kotlin/io/didomi/fluttersdk/DidomiPlugin.kt

```diff
@@ -318,6 +334,112 @@ class DidomiPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         }
     }
 
+    /**
+     * Get disabled purposes
+     */
+    private fun getDisabledPurposes(@NonNull result: Result) {
```

Comment:
I don't think @NonNull is useful here? We are in kotlin so Result is non-null by default